core-backend/wasmi/Cargo.toml

```diff
@@ -1,9 +1,13 @@
 [package]
 name = "gear-backend-wasmi"
 version = "0.1.0"
+description = "Gear core backend wasmi"
 authors.workspace = true
 edition.workspace = true
 license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+documentation.workspace = true
```

Comment:
We don't have documentation published for each package within the repo, recheck it please (and we don't need to)

Reply:
yeah make sense, if no documentation specified, `crates.io` will display the `docs.rs` one